SofaKernel/modules/SofaCore/src/sofa/core/State.h

```diff
@@ -105,6 +105,8 @@ class State : public virtual BaseState
     WriteOnlyVecDeriv writeOnlyVelocities() { return WriteOnlyVecDeriv(*this->write(core::VecDerivId::velocity())); }
     ReadVecDeriv  readForces() const        { return ReadVecDeriv (*this->read (core::ConstVecDerivId::force())); }
     WriteVecDeriv writeForces()             { return WriteVecDeriv(*this->write(core::VecDerivId::force())); }
+    ReadVecDeriv  readExtForces() const     { return ReadVecDeriv (*this->read (core::ConstVecDerivId::externalForce())); }
```

Comment:
I think this was deleted

Reply:
I couldn't find in the log. Are you sure?

Reply:
ok not enough arguments to keep that.